runtime/memory.cu

```diff
@@ -77,44 +77,44 @@ DEVICE_INLINE void adjustPartialLdMatrixAddrInTuring(unsigned& addr_in_byte) {
 // warp.
 
 template <typename T>
-DEVICE_INLINE void ldMatrix(Array<T, 4, 4>& out, unsigned addr) {
+DEVICE_INLINE void ldMatrix(Array<T, 4, 4>* out, unsigned addr) {
```

Comment:
Does this make any impact?

Reply:
I checked `ptxas_verbose`, the number of used registers are exactly the same. So I assume that if the existing SASS tests pass, then there shouldn't be anything wrong.